inspirehep/ext/jinja_filters/record.py

```diff
@@ -390,3 +390,99 @@ def citation_phrase(count):
             return 'Cited 1 time'
         else:
             return 'Cited ' + str(count) + ' times'
+
+    @app.template_filter()
+    def conference_info(record):
+        out = ''
+        title_exists = False
+        if 'publication_info' in record:
+            from invenio_ext.es import es
+            for element in record['publication_info']:
+                if 'journal_title' in element:
+                    title_exists = True
+                if 'conference_recid' in element \
+                        and 'parent_recid' in element:
+                    conference_rec = es.get(
+                        index='conferences', id=element['conference_recid'])
+                    if 'journal_title' not in element:
+                        if 'page_artid' in element:
+                            if title_exists:
+                                out += '(<a href="/record/' + str(
```

Comment:
Instead of having all the HTML here it can be nice adding all the snippets as macros in a Jinja template and then using this utility (https://github.com/inspirehep/invenio-ext/blob/master/invenio_ext/template/utils.py#L25) to render them.

See usage here https://github.com/inspirehep/invenio-deposit/blob/master/invenio_deposit/field_widgets.py#L554

( I like it because I wrote that helper :) )